platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Extensions/NSAttributedString+Attributes.swift

```diff
@@ -62,18 +62,27 @@ extension NSAttributedString {
     /// - Parameters:
     ///   - color: the new UIColor to update the attributed string
     ///   - linkColor: the new UIColor for links inside the attributed string
+    ///   - codeBackgroundColor: the new UIColor for the background of code blocks
     /// - Returns: a new attributed string with the same content and attributes, but its foregroundColor is changed
-    func changeColor(to color: UIColor, linkColor: UIColor) -> NSAttributedString {
+    func changeColor(to color: UIColor, linkColor: UIColor, codeBackgroundColor: UIColor) -> NSAttributedString {
         let mutableAttributed = NSMutableAttributedString(attributedString: self)
         mutableAttributed.addAttributes(
             [.foregroundColor: color], range: NSRange(location: 0, length: mutableAttributed.length)
         )
+        
         // This fixes an iOS bug where if some text is typed after a link, and then a whitespace is added the link color is overridden.
         mutableAttributed.enumerateAttribute(.link, in: NSRange(location: 0, length: mutableAttributed.length)) { value, range, _ in
             if value != nil {
                 mutableAttributed.addAttributes([.foregroundColor: linkColor], range: range)
             }
         }
+        
+        // Adding background to inline code
+        mutableAttributed.enumerateTypedAttribute(.font) { (font: UIFont, range, _) in
+            if font.familyName == "Courier" {
```

Comment:
Maybe something like `font.fontDescriptor.symbolicTraits.contains(.traitMonoSpace)` is a more fitting test for this. 

But this is one more reason to integrate DTCoreText inside the library to make customisation easier I guess.

Reply:
Wait I will check if I can make this work by using CSS styling. I think we also want to change the font, because I think we use Menlo-Regular but iOS by default parses Courier from HTML.

If this CSS styling doesn't work yeah we might need this workaround for now and in the future integrating DTCoreText would make things a lot easier and more customisable

Reply:
DONE!